sentry/src/main/java/io/sentry/Scope.java

```diff
@@ -1038,6 +1038,23 @@ public void setSpanContext(
     }
   }
 
+  @ApiStatus.Internal
+  @Override
+  public void replaceOptions(final @NotNull SentryOptions options) {
+    // TODO [HSM] check if already enabled and noop in that case?
```

Comment:
This shouldn't be used after `Sentry.init` since only some options are actually updated.